lib/iruby/chartkick/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module IRuby
   module Chartkick
-    VERSION = "0.3.1"
+    VERSION = "0.5.1"
```

Comment:
```suggestion
    VERSION = "0.4.0"
```
@rstammer what do you think? 🙂

Reply:
Satisfies my mathematical soul for coherent series a lot! I'm in for this 😁